source/Generic/PlayState/PlayState.csproj

```diff
@@ -68,7 +67,6 @@
     <None Include="extension.yaml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="packages.config" />
```

Comment:
Why was this removed?

Reply:
Was automatically removed by Visual Studio when changing to package reference: https://docs.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference

This change was necessary because it didn't compile after adding the notifications dependencies without doing so.